### PR TITLE
[PPD-47] add gpd service

### PR DESCRIPTION
**Pull request by pasqualespica**
The purpose of this PR is to create az resources for [µ-service: Gestione Posizioni Debitorie](https://pagopa.atlassian.net/wiki/spaces/PPD/pages/96436951/Design+Review+-+Posizioni+Debitorie#%C2%B5-servizio-1%3A-Gestione-Posizioni-Debitorie)

### List of changes

Added : 
  - `gdp` app service
  - publish `gpd` api on apim

### Motivation and context

According to [Gestione-Posizioni-Debitorie](https://pagopa.atlassian.net/wiki/spaces/PPD/pages/96436951/Design+Review+-+Posizioni+Debitorie#%C2%B5-servizio-1:-Gestione-Posizioni-Debitorie)

### Type of changes

- [x] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [x] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [x] No

### Other information

---

### CODE REVIEW

File: src/gpd.tf [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    name = "default"
+    service_delegation = {
+      name    = "Microsoft.Web/serverFarms"
+      actions = ["Microsoft.Network/virtualNetworks/subnets/action"]
+    }
+  }
+}
+
+module "gpd_app_service" {
+  source = "git::https://github.com/pagopa/azurerm.git//app_service?ref=v1.0.93"
```
Comment (pasqualedevita) [inline src/gpd.tf:35]: use latest app service modules.
When u create a new app service/function think if u can create a common app service plan to reduce costs

**Review by pasqualedevita — CHANGES_REQUESTED** [review]

File: src/env/uat/terraform.tfvars [hunk 2]
```diff
@@ -28,6 +28,8 @@ cidr_subnet_pagopa_proxy = ["10.1.240.24/29"]
 cidr_subnet_buyerbanks   = ["10.1.240.32/29"]
 
 cidr_subnet_reporting_fdr = ["10.1.240.40/29"]
+cidr_subnet_gpd           = ["10.1.240.48/29"]
```
Comment (pasqualedevita) [inline src/env/uat/terraform.tfvars:31]: why different subnet ranges in dev, uat, prod?

**Review by pasqualedevita — CHANGES_REQUESTED** [review]

File: src/env/uat/terraform.tfvars (hunk 2 above)
Comment (pasqualespica) [reply src/env/uat/terraform.tfvars:31]: we do it in to dedicated PR

File: src/gpd.tf (hunk 1 above)
Comment (pasqualespica) [reply src/gpd.tf]: done

File: src/env/uat/terraform.tfvars (hunk 2 above)
Comment (pasqualespica) [reply src/env/uat/terraform.tfvars]: aligned

**Pull request merged**